### DISCUSSION TITLE
traslation

### DISCUSSION THREAD

**Discussion by statick88** [quarto-dev/quarto-cli#1106] [Q&A] [answered]
Hello, I would like to contribute with the project, I know that my knowledge is not very advanced, but at least with the translation to "Spanish" I know that I could collaborate.

Is there any way to start the translation into this language?

**Comment by jjallaire** [comment, marked as answer]
We have a spanish translation for UI/content elements produced by Quarto here: https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/language/_language-es.yml. This is activated by adding `lang: es` to document options.

The Quarto CLI itself prints prompts, status messages, etc. in English and there isn't currently a localization framework for that (we aren't likely to do this soon as the higher priority is readers of content vs. users of the CLI tool).

**Comment by statick88** [comment]
If I imagined it, that's why I wanted to know about it, thank you very much for your answer, I hope to be able to contribute from another point of view.

Excellent week of work, and congratulations
